www/generate_tutorial/src/input/tutorial.md

```diff
@@ -398,6 +398,50 @@ The `fromScratch` and `fromOriginal` records are equal, although they're defined
 Note that `&` can't introduce new fields to a record, or change the types of existing fields.
 (Trying to do either of these will result in an error at build time!)
 
+## [Optional Record Fields](#optional-record-fields) {#optional-record-fields}
+
+Roc supports optional record fields using the `?` operator. This can be a useful pattern where you pass a function a record of configuration values, some of which you don't really care about and want to leave as defaults.
```

Comment:
Would rephrase to note that may care about them, but just provide default values (if you didn't care about them, why include them in the type at all?)

```suggestion
Roc supports optional record fields using the `?` operator. This can be a useful pattern where you pass a function a record of configuration values, some of which you'd like to provide defaults for.
```